TA-defender-atp-status-check/default/inputs.conf

```diff
@@ -1,10 +1,6 @@
 # Status of installation of Defender ATP
 [powershell://generate_defender_atp_status_logs]
 script = . "$SplunkHome\etc\apps\TA-defender-atp-status-check\bin\generate_defender_atp_status_logs.ps1"
-schedule = */30 * * * *
-disabled = 1
-
-[monitor://$SPLUNK_HOME\var\log\TA-defender-atp-status-check\DefenderATPStatus.log]
-disabled = 1
+schedule = 0 */6 * * *
```

Comment:
Can you please keep old schedule unless it is changed on purpose. Also put in seconds. Reason: If we mention cron, it won't run script on input enable/disable. but if we keep in seconds then it does run on input enable/disable.

Reply:
In the case of cron, I tested that the script will run on input enable/disable and then it will execute on the next cron schedule.
As discussed with Vatsal, the reason behind changing the schedule is to reduce the license usage (previously, the user has the option to ingest the event only if the status was updated)

Reply:
I doubt this: "In the case of cron, I tested that the script will run on input enable/disable"
* Please lets test this OR
* If not, then let's add and use the **interval** parameter to run on restart.

Reply:
I also tested and it's working. It's possible that Powershell input has different behaviour.

Reply:
Cool then, I'm okay with that.